src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
@vtjnash I'm working on a [header to help force-link GPURuntime](https://reviews.llvm.org/D33198). Please review the proposed changes to Julia in this [diff](https://github.com/JuliaLang/julia/compare/master...singam-sanjay:USE_POLLY_ACC-withGPURuntimeHeader). For some reason, I'm unable to merge these changes into the USE_POLLY_ACC branch, the branch in the pull request. I'll create another pull request using USE_POLLY_ACC-withGPURuntimeHeader in a while.